crates/hir_def/src/nameres/mod_resolution.rs

```diff
@@ -70,8 +71,13 @@ impl ModDir {
                 candidate_files.push(self.dir_path.join_attr(attr_path, self.root_non_dir_owner))
             }
             None => {
-                candidate_files.push(format!("{}{}.rs", self.dir_path.0, name));
-                candidate_files.push(format!("{}{}/mod.rs", self.dir_path.0, name));
+                if is_include_macro {
```

Comment:
Is is possible to move `file_id.is_include_macro(db.upcast());` inside here , such that :
```rust
if file_id.is_include_macro(db.upcast()) {
```

Reply:
Ok, I moved it inside.